exporter/collector/metrics.go

```diff
@@ -518,25 +518,43 @@ func (m *metricMapper) exemplars(exs pdata.ExemplarSlice) []*distribution.Distri
 // histogramPoint maps a histogram data point into a GCM point.
 func (m *metricMapper) histogramPoint(point pdata.HistogramDataPoint) *monitoringpb.TypedValue {
 	counts := make([]int64, len(point.BucketCounts()))
+	var mean, deviation, prevBound float64
+
 	for i, v := range point.BucketCounts() {
 		counts[i] = int64(v)
 	}
 
-	mean := float64(0)
 	if !math.IsNaN(point.Sum()) && point.Count() > 0 { // Avoid divide-by-zero
 		mean = float64(point.Sum() / float64(point.Count()))
 	}
 
+	bounds := point.ExplicitBounds()
+	// Calculate the sum of squared deviation.
+	for i, bound := range bounds {
+		// Assume all points in the bucket occur at the middle of the bucket range
+		middleOfBucket := (prevBound + bound) / 2
+		deviation += float64(counts[i]) * (middleOfBucket - mean) * (middleOfBucket - mean)
+		prevBound = bound
+	}
+	// The infinity bucket is an implicit +Inf bound after the list of explicit bounds.
+	// Make sure we have a count for that bucket, just in-case.
+	if len(counts) == len(bounds)+1 {
```

Comment:
Are there actually cases where the overflow bucket is not present? It doesn't appear to be optional in the [proto code](https://github.com/open-telemetry/opentelemetry-proto/blob/cef5447c443d558a4056a93a35d48b8a72e0fad6/opentelemetry/proto/metrics/v1/metrics.proto#L473). I believe Prometheus also requires +Inf to be present.

Reply:
Prom does require it.  I can remove it.  I just found it surprising, and didn't trust it.